boards/seeed/xiao_esp32s3/doc/index.rst

```diff
@@ -1,13 +1,28 @@
-.. zephyr:board:: xiao_esp32s3
+.. _xiao_esp32s3:
+
+XIAO ESP32S3 / XIAO ESP32S3 Sense
+####################################
```

Comment:
```suggestion
XIAO ESP32S3/XIAO ESP32S3 Sense
############################
```

Reply:
If you can fix this today - and we get a few more eyes on it - we may be able to get it in before the v4.0.0 feature freeze.

Reply:
Done, thanks.